conan/tools/google/bazeldeps.py

```diff
@@ -93,22 +94,52 @@ def _is_shared():
         return {"shared-library": True,
                 "static-library": False}.get(str(dep.package_type), default_value)
 
-    def _save_lib_path(lib_, lib_path_):
+    def _save_lib_path(file_name, file_path):
         """Add each lib with its full library path"""
-        formatted_path = lib_path_.replace("\\", "/")
-        _, ext_ = os.path.splitext(formatted_path)
-        if is_shared and ext_ == ".lib":  # Windows interface library
-            interface_lib_paths[lib_] = formatted_path
-        else:
-            lib_paths[lib_] = formatted_path
+        name, ext = file_name.split('.', maxsplit=1)  # ext could be .if.lib
```

Comment:
Dont we want ``name, ext = file_name.rsplit(".", 1)`` to get just the very last extension?

Reply:
Nope, I need the other one. The difference is:

```python
# libs = ["mylib"]
# mylib.if.lib
name, ext = file_name.rsplit(".", 1)  # mylib.if 
name, ext = file_name.split(".", 1)  # mylib -> I want this one
```

Reply:
The very last extension at this point is not important. The only important thing is to check if it's a `dll`.